### PR TITLE
Add view files for funnel data

### PR DESCRIPTION
**Pull request by ksiegler1**
Create view for 3 funnel datasets in BQ

- `moz-fx-data-shared-prod.accounts_frontend_derived.login_funnels_by_service_v1`
- `moz-fx-data-shared-prod.accounts_frontend_derived.registration_funnels_by_service_v1`
- `moz-fx-data-shared-prod.accounts_frontend_derived.email_first_reg_login_funnels_by_service_v1`

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/DENG-3708)

### CODE REVIEW

File: sql/moz-fx-data-shared-prod/accounts_frontend/email_first_reg_login_funnels_by_service/metadata.yaml
Comment (ksiegler1) [inline sql/moz-fx-data-shared-prod/accounts_frontend/email_first_reg_login_funnels_by_service/metadata.yaml:1]: @sean-rose I wasn't sure what to fill in (if anything) for the `metadata.yaml` files for the `labels`, `bigquery`, and `references` field.

Comment (sean-rose) [reply sql/moz-fx-data-shared-prod/accounts_frontend/email_first_reg_login_funnels_by_service/metadata.yaml:1]: You can safely remove those fields from these view metadata files.

**Review by sean-rose — APPROVED** [review]
The `deploy-changes-to-stage` CI job is failing because the `accounts_db.fxa_oauth_clients` view being referenced in these views is an authorized view, and `bqetl` doesn't currently publish authorized views to staging.  Once you're ready to merge let me know and I'll try manually running the CI with an argument to skip the `deploy-changes-to-stage` job.

**Comment by ksiegler1** [comment]
@sean-rose I am ready to merge!

**Comment by sean-rose** [comment]
> The `deploy-changes-to-stage` CI job is failing because the `accounts_db.fxa_oauth_clients` view being referenced in these views is an authorized view, and `bqetl` doesn't currently publish authorized views to staging.  Once you're ready to merge let me know and I'll try manually running the CI with an argument to skip the `deploy-changes-to-stage` job.

#5560 should fix the underlying problem with the `deploy-changes-to-stage` CI job, so once that's merged we can update this branch and then the CI checks should pass without having to skip the `deploy-changes-to-stage` job.

**Pull request merged**